README.md

```diff
@@ -63,7 +63,7 @@ function main(string... args) {
             io:println("Bucket Status: " + status);
         }
         //Unsuccessful attempts return an AmazonS3 error.
-        amazons3:AmazonS3Error e => io:println(e);
+        amazons3:error e => io:println(e);
```

Comment:
This should simply be the error right? Please check all the relevant places
error e => io:println(e);

Reply:
Fixed